taskcluster/scripts/toolchain/build-rust-toolchain.sh

```diff
@@ -30,6 +30,7 @@ rustup --version
 rustup target add x86_64-apple-darwin
 rustup target add x86_64-pc-windows-gnu
 rustup target add x86_64-unknown-linux-musl
+rustup target add aarch64-unknown-linux-gnu
 rustup target add aarch64-unknown-linux-musl
```

Comment:
Do we still support/need this?

Reply:
You added this in another patch, right? We can probably drop it if you're not going to use it. Once there are aarch64 native builders we should probably switch to compiling for a -musl target to be strictly more portable.